lib/train/extras/os_common.rb

```diff
@@ -96,6 +98,8 @@ def detect_family
       # TODO: extend base implementation for detecting the family type
       # to Windows and others
       case uname_s
+      when /unrecognized command verb/
```

Comment:
Its a pain because the uname_s command doesnt returns an error that can be detected by the command object (old os) but a message that says : **%DCL-W-IVVERB, unrecognized command verb - check validity and spelling**